docs/team/yourally2.md

```diff
@@ -0,0 +1,77 @@
+  of students in the class change. 
+    * Justification: The visual aspect allows the user to quickly tell which students belong to a class at a glance. It is also
+    intuitive to use as double clicking on a class to see the students is very natural. This forms the backbone of the class feature
+  as all interactions with the class is done in this tab.
+    * Highlights: A deep understanding of how the GUI functioned together with the different recommended design patterns was required.
+It was challenging to consider the flow of data for components while ensuring that the GUI turned out appealing.
+    * Pull Request: [\#103](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/103)
+
+* **New feature**: Add `class` feature to view a specific class without double clicking
+    * What it does: It shows the students belonging to the specified class. Does the same thing as double clicking on a class. 
+    * Justification: As Stuart mentioned, it is crucial because it follows the mindset of not touching the mouse or using voice commands. 
+    It allows us to perform the same functionality without using a mouse, which is important.
+    * Highlights: It was challenging to modify the GUI using a command, so after discussing with Stuart we decided to follow the Observer pattern,
+   which proved to be very useful.
+    * Pull Request (same PR as GUI): [\#103](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/103)
+ 
+* **Code contributed**: [Reposense](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=yourally2&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=yourally2&tabRepo=AY2122S1-CS2103T-F11-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+  
+* **Enhancements to existing features**:
+    * Refactor Person to Student and make Student and next-of-kin (NOK) inherit from Person (Pull Request [\#30](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/30))
+    * Wrote tests for `FindClassCommandParser` and `FindClassCommand` (Pull request [\#258](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/258)
+    * Bug fixes for GUI, `class` command and `Student list` in `classes` tab (Pull request 
+      [\#248](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/248),
+      [\#243](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/243))
+  
+* **Documentation**
+  *User Guide:
+    * Fixed various documentation issues and touched up on UG [\#224](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/224)
+  * Developer Guide
+    * Added user stories [\#22](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/22)
+    * Adding a class [\#86](https://github.com/AY2122S1-CS2103T-F11-1/tp/pull/86)
+  
+* **Community**
+  * PRs reviewed (with non-trivial review comments): (Pull request [\#26](),
```

Comment:
U have an extra pull request here #26 with no link. Do u want to remove it or add a link?